recipes/assertpy/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Let's remove the `build.sh` in favor of the `script` in the `meta.yaml` tag:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L29